### ISSUE TITLE
linux-cpu doesn't work

### ISSUE THREAD

**Issue by emdio** [CamWagner/lizzie#11]
Launching the cpu only version of lizzie.jar returns this error

$ java -jar lizzie.jar 
./leelaz: error while loading shared libraries: libboost_program_options.so.1.58.0: cannot open shared object file: No such file or directory
Leelaz process ended.
java.io.IOException: Stream closed
	at java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:433)
	at java.io.OutputStream.write(OutputStream.java:116)
	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:141)
	at wagner.stephanie.lizzie.analysis.Leelaz.sendCommand(Leelaz.java:137)
	at wagner.stephanie.lizzie.analysis.Leelaz.ponder(Leelaz.java:179)

The java version in my computer:
$ java -version
openjdk version "1.8.0_151"
OpenJDK Runtime Environment (build 1.8.0_151-8u151-b12-0ubuntu0.17.10.2-b12)
OpenJDK 64-Bit Server VM (build 25.151-b12, mixed mode)

**Comment by rainoffallingstar** [comment]
the same problem happens in my linux system ubuntu 17.10

**Comment by CamWagner** [comment]
Please follow the Linux installation instructions in https://github.com/gcp/leelazero. I will soon update Lizzie's Readme for Linux about this. You need Libboost, and maybe 1-2 other packages.

**Comment by emdio** [comment]
@CamWagner , the problem is that https://github.com/gcp/leela-zero doesn't provide instructions to compile a cpu only version of leela-zero (for whatever reason they just don't want to add them). In this issue are instructions on how to make a cpu only compilation.
https://github.com/gcp/leela-zero/issues/920

**Comment by emdio** [comment]
If I compile my own leelaz, the lizzie is launched correctly, but as I place stones on the board nothing happens. I use the space bar to switch pondering on and off, but no numbers appear in the board.
![lizzie](https://user-images.githubusercontent.com/1314942/37144511-92fbf59c-22be-11e8-95a5-e6905ad6a625.png)

**Comment by CamWagner** [comment]
Oh, but the one I included does not work after you've compiled that? Please make sure

**Comment by emdio** [comment]
Launching lizzie with the leelaz provided by you just returns the same error:

$ java -jar lizzie.jar 
./leelaz: error while loading shared libraries: libboost_program_options.so.1.58.0: cannot open shared object file: No such file or directory
java.io.IOException: Stream closed
Leelaz process ended.
	at java.lang.ProcessBuilder$NullOutputStream.write(ProcessBuilder.java:433)
	at java.io.OutputStream.write(OutputStream.java:116)
	at java.io.BufferedOutputStream.flushBuffer(BufferedOutputStream.java:82)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:140)
	at java.io.BufferedOutputStream.flush(BufferedOutputStream.java:141)
	at wagner.stephanie.lizzie.analysis.Leelaz.sendCommand(Leelaz.java:137)
	at wagner.stephanie.lizzie.analysis.Leelaz.ponder(Leelaz.java:179)
	at wagner.stephanie.lizzie.Lizzie.main(Lizzie.java:23)

If I make a "locate" of libboost this is what I get:

$ locate libboost_program_options.so
/usr/lib/x86_64-linux-gnu/libboost_program_options.so
/usr/lib/x86_64-linux-gnu/libboost_program_options.so.1.62.0

**Comment by hydrogenpi** [comment]
Same here.

**Comment by jfsamper** [comment]
Did you try sudo apt-get install libboost-all-dev ?

**Comment by CamWagner** [comment]
So it will not work even if you first run
    sudo apt install libboost-all-dev libopenblas-dev opencl-headers ocl-icd-libopencl1 ocl-icd-opencl-dev zlib1g-dev
? 

If so I will just release the modified files and Linux users can compile it according to the Leela Zero compilation steps.

**Comment by emdio** [comment]
No, it doesn't work either.

"If so I will just release the modified files and Linux users can compile it according to the Leela Zero compilation steps."
Maybe this is the best option. 
BTW, as seen in this issue, there are two ways to build a cpu-only version (commenting out a line of code or adding an option to "cmake"). 
https://github.com/gcp/leela-zero/issues/920
I wonder wich option you used for lizzie.

**Comment by emdio** [comment]
Ok, I see at issue #10   that unless you have the leelaz provided  by lizzie, you won't see nothing on your screen. If the provided leelaz doesn't work for linux, then that's it, you're stuck, aren't you?

**Comment by CamWagner** [comment]
It's fine I will just provide some code that Linux users should compile for Leela Zero. The instructions will be slightly longer for Linux but thats fine.

**Comment by pheasant75** [comment]
I have a ubuntu linux with GPU.  I tried the included GPU leelaz but it complain some cuda library issue.  Then I just replaced with the leelaz I compiled (and was running fine for many days).  The lizzie gui pops up, but only allow me to play with myself and the probability is not showing on the board, but it is printed in command window.
Also, is there any way to turn on the coordinates in the board (A B C D... and 1, 2, 3, 4 ...)?
